chatbot-core/requirements.txt

```diff
@@ -84,7 +84,7 @@ six==1.17.0
 sniffio==1.3.1
 soupsieve==2.7
 SQLAlchemy==2.0.41
-starlette==0.46.2
+starlette==0.49.1
```

Comment:
Conflicts

The conflict is caused by:
    The user requested starlette==0.49.1
    fastapi 0.115.12 depends on starlette<0.47.0 and >=0.40.0